feature-libs/user/profile/components/update-profile/update-profile.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component } from '@angular/core';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  RoutingService,
+} from '@spartacus/core';
+import { User } from '@spartacus/user/account/root';
+import { Title, UserProfileFacade } from '@spartacus/user/profile/root';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+@Component({
+  selector: 'cx-update-profile',
+  templateUrl: './update-profile.component.html',
+})
+export class UpdateProfileComponent {
+  titles$: Observable<Title[]> = this.userProfile.getTitles();
+  user$: Observable<User | undefined> = this.userProfile.get();
+  isLoading$ = new BehaviorSubject(false);
+
+  constructor(
+    private routingService: RoutingService,
+    private userProfile: UserProfileFacade,
+    private globalMessageService: GlobalMessageService
+  ) {}
+
+  onSuccess(): void {
+    this.globalMessageService.add(
+      { key: 'updateProfileForm.profileUpdateSuccess' },
+      GlobalMessageType.MSG_TYPE_CONFIRMATION
+    );
+    this.routingService.go({ cxRoute: 'home' });
+  }
+
+  onCancel(): void {
+    this.routingService.go({ cxRoute: 'home' });
+  }
+
+  onSubmit({ userUpdates }: { userUpdates: User }): void {
+    this.isLoading$.next(true);
+
+    this.userProfile.update(userUpdates).subscribe({
+      next: () => this.onSuccess(),
+      error: () => {},
```

Comment:
In this case shouldn't we also disable loading on error?

Reply:
We don't have an onError scenario here, will be fixed in the followup #11662